bindings/nodejs/lib/types/block/output/unlock-condition.ts

```diff
@@ -124,41 +125,44 @@ class StorageDepositReturnUnlockCondition extends UnlockCondition /*implements I
     }
 }
 /**
- * Timelock Unlock Condition.
+ * Defines a slot index until which the output can not be unlocked.
  */
-class TimelockUnlockCondition extends UnlockCondition /*implements ITimelockUnlockCondition*/ {
-    private unixTime: number;
+class TimelockUnlockCondition extends UnlockCondition {
+    private slotIndex: SlotIndex;
 
-    constructor(unixTime: number) {
+    constructor(slotIndex: SlotIndex) {
         super(UnlockConditionType.Timelock);
-        this.unixTime = unixTime;
+        this.slotIndex = slotIndex;
     }
     /**
-     * Unix time (seconds since Unix epoch) starting from which the output can be consumed.
+     * Slot index starting from which the output can be consumed.
      */
-    getUnixTime(): number {
-        return this.unixTime;
+    getSlotIndex(): SlotIndex {
```

Comment:
So here we are again, have a getter method or just make the field readable? :trollface:

Reply:
yea i saw it but thought maaaybe out of scope? :D ill update it all then